datalad/support/annexrepo.py

```diff
@@ -3297,12 +3311,23 @@ def get_content_annexinfo(
             path = self.pathobj.joinpath(ut.PurePosixPath(j['file']))
             rec = info.get(path, None)
             if rec is None:
-                if init is not None:
+                # git didn't report on this path
+                if j.get('success', None) is False:
```

Comment:
I guess should be of the same effect but the pattern we used already is 
```suggestion
                if not j.get('success', True):
```
which avoids `is`ing bools .